packages/components/src/components/DistributionsChart/utils.tsx

```diff
@@ -0,0 +1,46 @@
+import { SqScale, SqShape } from "@quri/squiggle-lang";
+/**
+ * A function to adjust the height of PDF values in accordance with non-linear scales.
+ * This is achieved by multiplying the values by the inverse of the derivative of the scale function at each point.
+ * The adjustment allows the PDF values to be more representative of the underlying data,
+ * providing a better fit for a histogram representation of the data with the given scales.
+ */
+function pdfScaleHeightAdjustment(
+  scale: SqScale
+): (x: number, y: number) => number {
+  switch (scale.tag) {
+    case "linear":
+      return (_, y) => y;
+    case "symlog":
+      return (x, y) => y * Math.abs(x);
```

Comment:
I'll change this once https://github.com/quantified-uncertainty/squiggle/pull/2086 is merged.